index.html

```diff
@@ -0,0 +1,311 @@
+              <h1 class="annual_music">Annual House Afrika Sessions 2021</h1>
+            </div>
+
+            <div class="aboutContent">
+                <p class="aboutInfo">Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                     Dolore rerum animi eveniet facilis reprehenderit est nemo quo totam quaerat
+                      repellendus expedita minima itaque, eligendi, 
+                      alias quidem non quia a facere!
+                </p>
+            </div>
+
+            <div class="contact">
+                <p>Please contact us via email for any question about the DeepHouse event</p>
+                <a class="contactEM" href="#">shimmy@DeepHouse.co.za</a>
+            </div>
+
+        </div>
+
+        <div class="Logo">
+            <div class="logoHeader"><h2>House Afrika<br> Music Sessions 2021</h2></div>
+
+            <div class="about-line">
+                
+            </div>
+
+            <div class="logo_hist">
+              <p class="about_logo">
+                  The logo belongs to the solisitc music logo
+              </p>
+            </div>
+
+            <div class="Logo-container">
+                <img class="logo_image" src="images/soulistic.jpg" alt="soulistic music logo">
+            </div>
+
+        </div>
+
+        <div class="about_line">
+        </div>
+
+        <div class="pastWrapper">
+          <div class="pastEvent">
+            <div class="pastTitle">
+                <h2 class="preHouseEvent">See the past House Afrika Sessions</h2>
+            </div>
+
+            <div class="pastInfo">
+                <p class="pastEventLogo">
+                    Take a look at the past events 
+                </p>
+            </div>
+          </div>
+       </div>
+
+       <div class="pastImages">
+         <div class="past_img">
+             <div class="overlay">
+               <h2 class="Year">2012</h2>
+               <h3 class="location">House Afrika session Johnessburg</h3>
+            </div>
+         </div>
+
+         <div class="past_img past_img_1 ">
+            <div class="overlay">
+              <h2 class="Year">2018</h2>
+              <h3 class="location">House Afrika session Limpopo</h3>
+           </div>
+        </div>
+       </div>
+
+       <div class="partner mobile-P">
+           <div class="heading">
+               <h1>
+                Partner
+               </h1>   
+           </div>
+
+           <div class="about-line">
+           </div>
+
+           <div class="firstLogos">
+               <div class="log"><img class="pImg" src="images/mozilla.png" alt="mozilla logo">
+               </div>
+               <div class="log">
+                   <img class="pImg" src="images/googlekorea.png" alt="google logo">
+               </div>
+               <div class="log">
+                   <img class="pImg" src="images/naver.png" alt="naver logo">
+               </div>
+           </div>
+       </div>
+    </section>
+
```

Comment:
- [ ] Kindly add the _about us_ page on a **different Html page** and provide the **navigation link** in the menu of the top white navbar